magley/sssscs
# Feat real certificates

How to test:
1. Run application
2. Create user with /register endpoint
3. Create root certificate (`creatorId = 1`, `parentId = null`)
4. It's auto-accepteed.
5. Check /keys and /certs folders in project directory.
6. Create intermediate certificate (`parentId = 1`)
7. Accept it
8. Experiment with different `validTo` values for when checking if it's valid (e.g. invalid parent should result in invalid child. Try at least a 3-level deep certificate chain).

When you're done, delete .cert and .key files (you don't have to, they'll be overridden next time).

Issues:

SerialNumber. How to generate a serial number to use as filename instead of raw id (idea: SHA256). Also, when and where to do this? Right now I do `cert.getId().toString()`, but the ID will be non-null after doing `certRepo.save(cert)` once, so creating a `Certificate` entity requires 2 database saves.
